.github/workflows/api-ci.yml

```diff
@@ -47,31 +47,6 @@ jobs:
       - name: Build everything
         run: pnpm build
 
-      - name: Check for incorrect or missing database migrations
```

Comment:
i put this into a script, added a "duplicate" drizzle prefix file check, and also moved it in the "code quality" action.